07-defensive.md

```diff
@@ -648,9 +648,18 @@ not more.
 > Explain in words what the assertions in this code check,
 > and for each one,
 > give an example of input that will make that assertion fail.
->  
+> 
 > ~~~ {.python}
 > def running(values):
+>     '''Given a sequence of values, return a list containing
+>        the running totals at each index.
+>
+>        >>> running([1, 2, 3, 4])
```

Comment:
Learners won't have seen doctests, so this will confuse them more than enlighten.